crates/biome_html_parser/tests/html_specs/ok/doctype/after-processing-instruction.html

```
@@ -0,0 +1,3 @@
+<?xml version="1.0"?>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)